docs/content/latest/api/ysql/extensions.md

```diff
@@ -274,14 +274,19 @@ The simplest way to set it up locally is to install it together with regular Pos
     $ brew install postgres && brew install postgis
     ```
 
-#### Ubuntu 18.04
-On Ubuntu, you can add the [PostgreSQL APT sources](https://www.postgresql.org/download/linux/ubuntu/). Then you can 
-install:
+#### Ubuntu
+Add the [PostgreSQL APT sources](https://www.postgresql.org/download/linux/ubuntu/). Then you can install:
 
 ```shell script
 sudo apt-get install postgresql-11 postgresql-11-postgis-3
 ```
 
+#### Centos
+You can get the YUM repository from [PostgreSQL website](https://www.postgresql.org/download/linux/redhat/). Then you 
+can use `yum` or `dnf` to install:
+```shell script
+yum install postgresql-11 postgresql-11-postgis-3
```

Comment:
Need sudo and package names are not correct for centos (using repo from PostgreSQL site):
  sudo yum install postgresql11-server postgis31_11